zulint/linters.py

```diff
@@ -3,7 +3,9 @@
 
 import argparse
 import subprocess
-if False:
+
+MYPY = False
+if MYPY:
     # See https://zulip.readthedocs.io/en/latest/testing/mypy.html#mypy-in-production-scripts
```

Comment:
The information that this linked to was removed as no longer relevant (zulip/zulip@a109508e3463dc3e3d8ece332f42a4ca5e4f1adc). I’d just remove this comment; `if MYPY` is clear enough.

Reply:
Pushed with this change in a new commit.